src/cephadm/cephadm

```diff
@@ -2112,9 +2102,6 @@ def command_deploy():
                       osd_fsid=args.osd_fsid,
                       reconfig=args.reconfig)
 
-        if crash_keyring and not args.reconfig:
```

Comment:
Should we clean-up `get_config_and_both_keyrings` to remove the `crash_keyring` now that it's no longer passed via the json?

Reply:
And also remove the obsolete `--crash-keyring`arg .. ?